ops/prod/main.tf

```diff
@@ -55,6 +55,12 @@ resource "azurerm_storage_queue" "fhir_publishing_error_queue" {
   storage_account_name = azurerm_storage_account.app.name
 }
 
+resource "azurerm_storage_share" "db_client_export" {
+  name                 = "db-client-export-${local.env}"
+  storage_account_name = azurerm_storage_account.app.name
+  quota                = 10
```

Comment:
(non-blocking question) It's unclear to me from reading the docs: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/storage_share#quota -- is this the max size of one file that is shared or the total size we have available to us?

Reply:
My understanding is that this number indicates that we have 10GB available to us from this volume mount at `/export`